<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica-Normal</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EYEGUARD</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);</string>
  </property>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_start
{
color: #808080;
background-color:#e6e6e6;
border-radius: 20px;
border:1px solid  #808080;
}

#pushButton_start:hover{
color: #2f8dff;
border:1px solid  #2f8dff;

}
</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,170);</string>
   </property>
   <widget class="QPushButton" name="pushButton_time">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_time{
background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-03.png);
}

#pushButton_time:hover{
border-image: url(icon/icon_blue-03.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_blink">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_blink{
background-color:transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-04.png);
}

#pushButton_blink:hover{
border-image: url(icon/icon_blue-04.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_distance">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_distance{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-06.png);
}

#pushButton_distance:hover{
border-image: url(icon/icon_blue-06.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_angle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_angle{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-05.png);
}

#pushButton_angle:hover{
border-image: url(icon/icon_blue-05.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sex">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_sex{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-07.png);
}

#pushButton_sex:hover{
border-image: url(icon/icon_blue-07.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_age">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_age{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-08.png);
}

#pushButton_age:hover{
border-image: url(icon/icon_blue-08.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bright">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>320</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_bright{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-09.png);
}

#pushButton_bright:hover{
border-image: url(icon/icon_blue-09.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>30</width>
      <height>1</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{border-color: rgb(255, 255, 255);
border:1px solid  #ffffff;

}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>30</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
border:1px solid  #ffffff;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_time_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_time_2{
background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-03.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_blink_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_blink_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-04.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bright_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>320</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_bright_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-09.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_age_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_age_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-08.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sex_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_sex_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-07.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_distance_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_distance_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-06.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_angle_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_angle_2{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-05.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_time</zorder>
   <zorder>pushButton_blink</zorder>
   <zorder>pushButton_distance</zorder>
   <zorder>pushButton_angle</zorder>
   <zorder>pushButton_sex</zorder>
   <zorder>pushButton_age</zorder>
   <zorder>pushButton_bright</zorder>
   <zorder>line_2</zorder>
   <zorder>line</zorder>
   <zorder>pushButton_time_2</zorder>
   <zorder>pushButton_blink_2</zorder>
   <zorder>pushButton_bright_2</zorder>
   <zorder>pushButton_age_2</zorder>
   <zorder>pushButton_sex_2</zorder>
   <zorder>pushButton_distance_2</zorder>
   <zorder>pushButton_angle_2</zorder>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>150</width>
     <height>130</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_menu{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-02.png);

}
#pushButton_menu:hover{
border-image: url(icon/icon_blue-02.png);
background-color:rgba(250, 250, 250,150);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu_active">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_menu_active{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_blue-02.png);

}
#pushButton_menu_active:hover{
background-color:rgba(250, 250, 250,150);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closs">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#pushButton_closs{
background-color: transparent ;
color: rgb(255, 255, 255);
border-image: url(icon/icon_white-10.png);

}
#pushButton_closs:hover{
border-image: url(icon/icon_blue-10.png);
background-color:rgba(250, 250, 250,150);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_image</zorder>
  <zorder>widget_bar</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_menu</zorder>
  <zorder>pushButton_menu_active</zorder>
  <zorder>pushButton_closs</zorder>
 </widget>
 <resources>
  <include location="icon_eye.qrc"/>
 </resources>
 <connections/>
</ui>
